examples/exampleChromeDriver/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "chromedriver-example",
+  "version": "0.1.0",
+  "description": "Chromedriver & Nightwatch ReportPortal example.",
+  "scripts": {
+    "test:chromedriver": "nightwatch --config ./nightwatch.json --reporter ./reporter.js"
+  },
+  "devDependencies": {
```

Comment:
I think it can be placed in the root package.json for now. One `npm install` to run all examples

Reply:
Fixed